src/store/Transaction.ts

```diff
@@ -105,10 +158,11 @@ export default {
     getters: {
         getInitialized: (state: TransactionState) => state.initialized,
         isFetchingTransactions: (state: TransactionState) => state.isFetchingTransactions,
-        confirmedTransactions: (state: TransactionState) => state.confirmedTransactions,
```

Comment:
If the getter get removed, do we still need the mutation below?

Reply:
yes, we are still using group filtering here